src/docs/markdown/automatic-https.md

```diff
@@ -220,3 +222,83 @@ Before attempting any ACME transactions, Caddy will test the configured storage
 Caddy can obtain and manage wildcard certificates when it is configured to serve a site with a qualifying wildcard name. A site name qualifies for a wildcard if only its left-most domain label is a wildcard. For example, `*.example.com` qualifies, but these do not: `sub.*.example.com`, `foo*.example.com`, `*bar.example.com`, and `*.*.example.com`.
 
 To get a wildcard from Let's Encrypt, you simply need to enable the [DNS challenge](#dns-challenge) and use a wildcard domain in your config. We recommend using wildcards only when you have so many subdomains that you would encounter CA rate limits trying to obtain certificates for them all.
+
+
+
+## Examples
+
+Caddy implicitly uses the HTTPS port (default 443) for your [server addresses](/docs/conventions#network-addresses) that don't assign an explicit port (which would disable automatic HTTPS). 
+
+The global setting `auto_https` has two values:
+- `disable_redirects` adds an implicit HTTP port redirect.
+- `off` disables automatic HTTPS, default implicit port changes to HTTP for all server addresses.
+
+| auto_https        | HTTP        | HTTPS       |
+|-------------------|-------------|-------------|
+| *default*         | redirects   | connects    |
+| disable_redirects | unreachable | connects    |
+| off               | connects    | unreachable |
+
+`unreachable` is the default without additional configuration when using the related `auto_https` setting.
+`connects` is the implicitly assigned port.
+
+
+### Disabling automatic HTTPS
+
+For local development environments, you can prevent serving via HTTPS by providing an explicit port assignment(disable per server address) or using the global `auto_https off` setting which will change Caddy's implicit port to be the HTTP port (default 80) globally.
+
+```caddy
+{
+  auto_https off
+}
+
+# Have caddy implicitly use the HTTP port
```

Comment:
I think the comments should be hoisted out of the code blocks with a quick line right above them that describe what follows. Reads better to have the purpose of the example just before it. Also reads better to have a line of text between each example code block.